sdk/graphql-transport/src/mappers/move.ts

```diff
@@ -156,6 +157,7 @@ export function mapNormalizedMoveModule(
 ): SuiMoveNormalizedModule {
 	const exposedFunctions: Record<string, SuiMoveNormalizedFunction> = {};
 	const structs: Record<string, SuiMoveNormalizedStruct> = {};
+	const enums: Record<string, SuiMoveNormalizedEnum> = {}; // TODO: Implement
```

Comment:
It looks like enums are actually available in the GraphQL api now, but this requires the graphql client to be updated.  The GraphQL release process has changed, and updating the client will take a bit of additional work.  So this should be fine for now, until we get the client updated